sam.local.yml

```diff
@@ -0,0 +1,29 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: 'is-research'
+
+Resources:
+  IsResearch:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: app.handle_event
+      Events:
+        Api:
+          Type: Api
+          Properties:
+            Path: /api/v0.1/items/{nypl_source}/{id}/is-research
+            Method: get
+            Request:
+              Parameters:
+                Paths:
+                  Nypl_source: true
+                  Id: true
```

Comment:
This is how we get the nypl_source and id from the URL. It can just be ``Request > Parameters > ..``. True means it is required. Right now, if one or both parameters are left out we get ``"message": "Missing Authentication Token"``. Tried changing true to false and still the same. Not sure how to set a custom error message, because handle_event does not get hit if one or both params are missing, e.g. `api/v0.1/items/blah/is-research` or `api/v0.1/items/is-research`

Reply:
It doesn't seem to be necessary for me.. Let's take a look at why it might be necessary on your system.